mysql/assets/SERVICE_CHECK_CLARIFICATION.md

```diff
@@ -0,0 +1,13 @@
+## On a slave host:
```

Comment:
We also return OK if we couldn't detect anything yet AND `Slave_running` `on` - https://github.com/DataDog/integrations-core/blob/master/mysql/datadog_checks/mysql/mysql.py#L583